### PR TITLE
Flesh out instructions for working with development version

### PR DESCRIPTION
**Pull request by martyngigg**
Added clarification on the steps to run on Linux to get setup with Conda to build and run the tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,49 +4,103 @@ This package provides a convenient set of Python wrappers
...
+
+We currently rely on an external fortran compiler as the current code is sensitive
+to the compiler version. Download the following files from the
+[tdm-gcc](https://sourceforge.net/projects/tdm-gcc/files/TDM-GCC%20Old%20Releases/TDM-GCC%204.6%20series/4.6.1-tdm64-1/):
+
+- gcc-4.6.1-tdm64-1-core.zip
+- gcc-4.6.1-tdm64-1-fortran.zip
+
+Once downloaded extract each of the files to a directory, e.g. `C:\MinGW64\4.6.1`, and add this
+directory to your `PATH` environment variable (you will need to restart any prompts open).
```
Comment (AnthonyLim23) [inline README.md:74]: which path environment variable is this? Does it mean in git or conda?

File: README.md [hunk 2]
```diff
@@ -4,49 +4,103 @@ This package provides a convenient set of Python wrappers
...
 
-To build a wheel, run
+Create a minimal conda environment from the provided environment file:
 
+```bat
+> mamba env create -f quasielasticbayes-dev-win.yml
```
Comment (AnthonyLim23) [inline README.md:79]: just tried these commands in both powershell and git bash, but neither work.

Comment (martyngigg) [reply README.md:79]: Did mambaforge install correctly using the above links? What does `where mamba` say?

Comment (AnthonyLim23) [reply README.md:79]:
``` The term 'mamba' is not recognized as the name of a cmdlet, function, script file, or operable program. Check
the spelling of the name, or if a path was included, verify that the path is correct and try again.```

Comment (AnthonyLim23) [reply README.md:79]: image
looks like its installed

Comment (martyngigg) [reply README.md:79]: Maybe I modified my path then...

Try adding `C:\Users\BTR75544\AppData\Local\mambaforge\condabin` to the `PATH` environment variable along with the `MinGW` directory...

**Comment by AnthonyLim23** [comment]
for windows I had to use these instructions https://www.mantidproject.org/Building_With_MinGW_On_Win64.html

**Comment by martyngigg** [comment]
@AnthonyLim23 I've added those instructions from the wiki. Let me know if there is anything else you think needs doing.

File: README.md [hunk 3]
```diff
@@ -4,49 +4,102 @@ This package provides a convenient set of Python wrappers
...
+Download and install the latest version of `mamba` for your platform from the link above by running the script you download and following the instructions.
 
+### Linux
+
+Create a minimal conda environment named `quasielasticbayes-dev` (you can use another name if you prefer):
+
+```sh
+> mamba env create -f quasielasticbayes-dev-win.yml
+> conda activate quasielasticbayes-dev
+> which python  # should produce something in $HOME/mambaforge
```
Comment (AnthonyLim23) [inline README.md:34]: this command does not seem to exist on my windows machine

File: README.md [hunk 4]
```diff
@@ -4,49 +4,102 @@ This package provides a convenient set of Python wrappers
...
+### Windows
+
+We currently rely on an external fortran compiler, `tdm64-gcc 4.6.1`, as the current code is sensitive
+to the compiler version. To install:
 
-To build a wheel, run
+- Download [tdm64-gcc-4.6.1.exe](https://sourceforge.net/projects/tdm-gcc/files/TDM-GCC%20Installer/Previous/1.1006.0/tdm64-gcc-4.6.1.exe/download) and install it to ``C:\MinGW64``
+- Download [gcc-4.6.1-tdm64-1-fortran.zip](https://sourceforge.net/projects/tdm-gcc/files/TDM-GCC%20Old%20Releases/TDM-GCC%204.6%20series/4.6.1-tdm64-1/gcc-4.6.1-tdm64-1-fortran.zip/download)
+- Right-click on ``gcc-4.6.1-tdm64-1-fortran.zip``, select "Extract All" and enter the location as ``C:\MinGW64``
+- Add ``C:\MinGW64\bin`` to your ``PATH`` environment variable ([instructions here](http://geekswithblogs.net/renso/archive/2009/10/21/how-to-set-the-windows-path-in-windows-7.aspx))
```
Comment (AnthonyLim23) [inline README.md:72]: the link is dead

Comment (AnthonyLim23) [reply README.md:72]: Also you need to add it (even if you already have it by removing it first) after making the environment.

Comment (martyngigg) [reply README.md]: Thanks. I've updated the link.

What did you mean by the second comment. Does just restarting the terminal after adding the environmental variable not do the trick?

Comment (martyngigg) [reply README.md]: Oh I see. That's not what I saw. I will add a paragraph that says if you run into trouble try this.

**Review by AnthonyLim23 — APPROVED** [review]
All looks good to me

**Pull request merged**